src/Models/Site.php

```diff
@@ -270,7 +270,7 @@ public function getPlan()
         if (empty($this->plan)) {
             $nickname = \uniqid(__FUNCTION__ . "-");
             $this->getContainer()->add($nickname, Plan::class)
-                ->addArgument(['site' => $this]);
+                ->addArguments([null, ['site' => $this]]);
```

Comment:
Is this broken on Terminus 2? Good catch.

Reply:
@greg-1-anderson No, this is a 3x-related bug